lib/ui/chat/widgets/chat_search_widget.dart

```diff
@@ -139,7 +139,7 @@ class _ChatSearchWidgetState extends ConsumerState<ChatSearchWidget> {
                       height: 16.w,
                       width: 16.w,
                       colorFilter: ColorFilter.mode(
-                        context.colors.primarySolid,
+                        context.colors.solidPrimary,
```

Comment:
Just checked, solidPrimary and primarySolid are both the same value in their light and dark modes. If this is intended as a cleanup, please remove one of them from the appTheme, probably primarySolid (and please help correct all the instances). Thanks.

Reply:
Good catch!